airbyte-integrations/connectors/source-klaviyo/source_klaviyo/spec.json

```diff
@@ -19,6 +19,13 @@
         "examples": ["2017-01-25T00:00:00Z"],
         "type": "string",
         "format": "date-time"
+      },
+      "events_look_back_window": {
```

Comment:
instead of having this as a parameter for user to update, should we add a constant value for this ?

Reply:
Yeah, probably better to keep as constant. Since this is not a standard practice for klaviyo but kind of our implmentation, not sure if the user will have enough information for them to select a value.